src/main/kotlin/blackjack/controller/BlackjackController.kt

```diff
@@ -0,0 +1,42 @@
+package blackjack.controller
+
+import blackjack.domain.Blackjack
+import blackjack.domain.Player
+import blackjack.service.InputParser
+import blackjack.view.BlackjackView
+
+class BlackjackController {
+
+    fun start() {
+        BlackjackView.printPlayerInput()
+        val players = getPlayers()
+        BlackjackView.printInitialize(players.joinToString(", ") { it.name })
+
+        val blackjack = Blackjack(players)
+
+        players.forEach { blackjack.drawingCards(it, 2) }
+        players.forEach { BlackjackView.printPlayerCard(it.name, it.cards.joinToString()) }
+        players.forEach { drawCard(blackjack, it) }
+        players.forEach { BlackjackView.printResult(it.name, it.cards.joinToString(), blackjack.calculatePoints(it)) }
+    }
+
+    private fun getPlayers(): List<Player> {
+        return InputParser.parsePlayerName(readLine()!!).map(::Player)
+    }
+
+    private fun drawCard(blackjack: Blackjack, player: Player) {
+        BlackjackView.printMoreCard(player.name)
+        while (InputParser.parseMoreCard(readLine()!!)) {
```

Comment:
```suggestion
        while (InputParser.parseMoreCard(readln())) {
```
- 코틀린 1.6부터는 조금 더 세련된(?) 읽기 API를 제공합니다.
- 값을 읽어오는 로직들은 InputView로 분리해도 괜찮을 것 같아요.